### ISSUE TITLE
Not support keyword 'super'

### ISSUE THREAD

**Issue by himulawang** [Polymer/hydrolysis#129]
```javascript
/**
 * Created by ila on 5/9/2015.
 */
"use strict";
class RedisProfile extends Profile {
    constructor(arr) {
        super(arr);

        this.type = 'Redis';
        if (arr !== undefined) this.fromArray(arr);
    }

    get db() { return this._db; }

    set db(v) { this._db = v; }

    toArray() {
        let arr = super.toArray();
        arr['db'] = this.db;
        return arr;
    }

    fromArray(arr) {
        this.db = arr.db;
        super.fromArray(arr);
    }
}
```

output
```
D:\Open Source\observer>vulcanize index.html --inline-script   | crisper --html build.html --js build.js
[Error: Error parsing script in index.html at NaN:NaN
undefined]
Error: Error parsing script in index.html at NaN:NaN
undefined
    at Analyzer._processScript (C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize\node_modules\hydrolysis\lib\analyzer.js:298:13)
    at Analyzer.<anonymous> (C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize\node_modules\hydrolysis\lib\analyzer.js:326:19)
```

If I delete super keyword. It's OK.

**Comment by azakus** [comment]
Duplicate of #82 
Fixed by #127 in the next release `1.14.0`.

**Comment by himulawang** [comment]
still has this problem

···
C:\Users\ila>npm install -g vulcanize
C:\Users\ila\AppData\Roaming\npm\vulcanize -> C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize\bin\vulcanize
vulcanize@1.9.3 C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize
├── path-posix@1.0.0
├── es6-promise@2.3.0
├── dom5@1.1.0 (parse5@1.4.2)
├── nopt@3.0.2 (abbrev@1.0.7)
├── hydrolysis@1.15.0 (estraverse@3.1.0, setimmediate@1.0.2, espree@2.0.3, doctrine@0.6.4)
└── update-notifier@0.5.0 (is-npm@1.0.0, string-length@1.0.0, semver-diff@2.0.0, repeating@1.1.3, chalk@1.0.0, configstore@1.1.0, latest-version@1.
0.1)

C:\Users\ila>vulcanize -v
vulcanize: 1.9.3
···

···
D:\Open Source\observer>vulcanize index.html --inline-script   | crisper --html build.html --js build.js
[Error: Error parsing script in index.html at NaN:NaN
undefined]
Error: Error parsing script in index.html at NaN:NaN
undefined
    at Analyzer._processScript (C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize\node_modules\hydrolysis\lib\analyzer.js:298:13)
    at Analyzer.<anonymous> (C:\Users\ila\AppData\Roaming\npm\node_modules\vulcanize\node_modules\hydrolysis\lib\analyzer.js:326:19)
···

Seems keyword 'super' cause this error.
